### PR TITLE
ENT-7246: Added documentation for new Augments features

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

File: reference/language-concepts/augments.markdown [hunk 1]
```diff
@@ -30,6 +32,112 @@ Any filenames you put here will appear in the `def.augments_inputs` variable.
 The standard set of masterfiles refers to this variable and will autoload those
 files.
 
+## variables
+
+Variables defined here can target a _namespace_ and or _bundle_ scope explicitly. When defined from `host_specific.json`, variables default to the `main` _bundle_ in the `data` __namespace_ (`$(data:main.MyVariable)`).
```
Comment (vpodzime) [inline reference/language-concepts/augments.markdown:37]: Variables defined in `vars` can also target a namespace and/or a bundle scope explicitly. `variables` only adds the possibility to specify tags and comment. Also the default namespace:bundle for variables is `default:def` and `data:variables` for `def.json` and `host_specific.json`, respectively.

File: reference/language-concepts/augments.markdown [hunk 2]
```diff
@@ -30,6 +32,112 @@ Any filenames you put here will appear in the `def.augments_inputs` variable.
...
+```
+
+Variables can also target a namespace explicitly.
+
+For example:
+
+```json
+{
+    "variables": {
+        "MyNamespace:my_bunlde.Variable": "value",
```
Comment (vpodzime) [inline reference/language-concepts/augments.markdown:72]: `bundle`

**Review by vpodzime — COMMENTED** [review]
Looks good to me otherwise.

File: reference/language-concepts/augments.markdown (hunk 1 above)
Comment (nickanderson) [reply reference/language-concepts/augments.markdown:37]: Thank you. updated the vars section.

File: reference/language-concepts/augments.markdown (hunk 2 above)
Comment (nickanderson) [reply reference/language-concepts/augments.markdown]: thanks

**Review by vpodzime — COMMENTED** [review]
Looks good to me so far.

**Review by vpodzime — APPROVED** [review]
ACK. Please squash the commits

**Review by vpodzime — APPROVED** [review]
Nice! Squash and merge!

File: reference/language-concepts/augments.markdown [hunk 3]
```diff
@@ -6,45 +6,172 @@ sorting: 70
...
+Variables defined here can target a _namespace_ and or _bundle_ scope explicitly. When defined from `host_specific.json`, variables default to the `main` _bundle_ in the `data` __namespace_ (`$(data:main.MyVariable)`).
+
+For example:
+
+```json
+{
+    "variables": {
+        "VariableWithImplicitNamespaceAndBundle": "value",
+        "comment": "An optional note about why this variable is important",
+        "tags": [ "optional", "tags" ]
```
Comment (olehermanse) [inline reference/language-concepts/augments.markdown:52]: @nickanderson @craigcomstock @vpodzime This is not how it works, right? AFAIK `comment` and `tags` need to be inside an object.

**Pull request merged**